### PR TITLE
Timezone requirements for 4.14+

### PR DESCRIPTION
**Pull request by andrijapanicsb**
Note on explicit Timezone requirements due to my mysql-connector 8.x java dependency

### CODE REVIEW

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
requesting docbuild

File: source/upgrading/upgrade/_timezone.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+You can do this by editing the /etc/cloudstack/management/db.properties file and adding "serverTimezone=UTC"
+to the "db.cloud.url.params=" and "db.usage.url.params=" lines.  Example lines, from a clean 4.14 installation, are given below:
+
+.. parsed-literal::
+
+   db.cloud.url.params=prepStmtCacheSize=517&cachePrepStmts=true&sessionVariables=sql_mode='STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION'&serverTimezone=UTC
+   db.usage.url.params=serverTimezone=UTC
+
+If you want time stamps in DB to use a different time zone, please make sure to test this thoroughly in test environment.
```
Comment (DaanHoogland) [inline source/upgrading/upgrade/_timezone.rst:33]: `If you want time stamps in the DB to use a different time zone, please make sure to test this thoroughly in a test environment.`

Comment (andrijapanicsb) [reply source/upgrading/upgrade/_timezone.rst]: Removed the whole line, as even when changing this from UTC to Europe/Belgrade (CEST) - nothing happens, no dates are changed in DB, so this atm can NOT be changed at all.

**Pull request merged**